Basic/APIDiorama/Packages/manifest.json

```diff
@@ -0,0 +1,50 @@
+{
+  "dependencies": {
+    "com.unity.feature.development": "1.0.1",
+    "com.unity.ide.rider": "3.0.18",
+    "com.unity.ide.visualstudio": "2.0.17",
+    "com.unity.ide.vscode": "1.2.5",
+    "com.unity.inputsystem": "1.4.4",
+    "com.unity.learn.iet-framework": "3.1.1",
+    "com.unity.learn.iet-framework.authoring": "1.2.1",
+    "com.unity.multiplayer.samples.coop": "https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop.git?path=/Packages/com.unity.multiplayer.samples.coop",
+    "com.unity.multiplayer.tools": "1.0.0",
+    "com.unity.netcode.gameobjects": "1.2.0",
+    "com.unity.render-pipelines.universal": "12.1.7",
+    "com.unity.test-framework": "1.1.33",
+    "com.unity.textmeshpro": "3.0.6",
+    "com.unity.ugui": "1.0.0",
+    "com.veriorpies.parrelsync": "https://github.com/VeriorPies/ParrelSync.git?path=/ParrelSync#6be2a26245e29070bf589809f10637af43570050",
```

Comment:
I'm a bit concerned by the fact that we prefer using ParrelSync instead of our very own Multiplayer PlayMode from the Tools team.

If a dev were to have MP PM installed and being able to run multiple instances for testing under the same Editor instance, would they need ParrelSync anymore? I personally do not think so, and this is a great opportunity (new greenfield sample project) to be on our (Unity's, Tools Team's) side and make use of our own tech.

Reply:
@becksebenius-unity FYI just in case if you'd like to participate.

Reply:
Just for awareness, mppm requires Unity 2023, our production readiness samples use LTS only.
Even for future bitesize work that'd be based on experimental dedicated server tech, mppm doesn't support dedicated server platform yet so we'd need to rely on ParrelSync at least hybrid with mppm

Reply:
Hi @ThusSpokeNomad !

> I'm a bit concerned by the fact that we prefer using ParrelSync instead of our very own Multiplayer PlayMode from the Tools team.

It's not a matter of preference: MPPM is available only for Unity 2023 onward. This sample's target version is Unity 2021 LTS. We can't use MPPM here.

When 2023 will be out, we'll be able to use MPPM instead of a 3rd party solution